docs/data-collection/intro.md

```diff
@@ -75,45 +80,95 @@ The above graph can be broken down as follows:
+    The ET arm is installed inside the Scanning Room, inside the scanner's bore, standing on a transparent plexiglass panel tailored to the scanner's bed rail.
+    Two lengthy cables (black for power and orange for signal) are passed through the access panel and connect the ET arm to the ET computer.
+* The ET computer is a standard PC with customized software for the control of the ET.
+* The infrared mirror is critical to be able to record the **right eye** of the participant, and is mounted covering the standard mirror of the coil.
 
-    It is critical to familiarize with the [GA's manual](../assets/files/GA_manual.pdf) to learn about its correct utilization.
+## Scanning protocols
 
-    !!! danger "Make sure you understand the switching on and off procedures described in these SOPs"
+The study will collect data with two different scanning protocols.
+Most of what is described in the present SOPs addresses the *Reliability Imaging Protocol* that will be acquired on the BH07060 scanner.
 
-    The input to the GA must be connected through a DM-060-24 desiccant chamber and a MLA0110 flow valve and a MLA0343 drying tube:
+| **Reliability Imaging Protocol (36 sessions $\times$ one scanner)** | mm:ss | **Standard Imaging Protocol (12 sessions $\times$ three scanners)** | mm:ss |
+|---|---|---|---|
+| Head scout | 00:14 | Head scout | 00:14 |
+| FoV Manual positioning | 01:00 | FoV Manual positioning | 01:00 |
+| T1w (anatomical reference) | 5:41 | T1w (anatomical reference) | 5:41  |
+| DWI (dMRI, structural connectivity) | 33:52 | EPI BOLD ($B_\text{0}$ field mapping) | 00:30 |
+| DWI ($B_\text{0}$ field mapping) | 00:54 | BOLD (single-echo RSfMRI, eyes open) | 20:09 |
+| GRE ($B_\text{0}$ field mapping) | 02:38 | GRE ($B_\text{0}$ field mapping) | 03:08 |
+| EPI BOLD ($B_\text{0}$ field mapping) | 00:27 | BOLD (multi-echo RSfMRI, eyes open, **only BH07060**) | 10:09 |
+| BOLD (positive-control task fMRI) | 03:07 | DWI (dMRI, structural connectivity) | 27:31 |
+| BOLD (multi-echo RSfMRI, naturalistic movie) | 20:09 | DWI ($B_\text{0}$ field mapping) | 01:10 |
+| BOLD (breath-holding task fMRI) | 06:00  | T2w (anatomical reference) | 08:24 |
+| T2w (anatomical reference) | 05:10  | | |
+| **Total Acquisition Time** | 79:12 | **Total Acquisition Time** | 77:56 |
 
-    ![desiccant_chamber](../assets/images/desiccant_chamber.png)
+!!! important "Stimulation program timings"
 
-    !!! warning "The MLA0343 drying tube and the DM-060-24 desiccant chamber MUST be replaced when their inside color turns into pink."
+    <a name="task-timing"></a>
+    The stimuli presentation laptop ({{ secrets.hosts.psychopy | default("███") }}) will execute four experiments, which will allow the synchronization of all devices by sending the adequate signals at their predesignated times and also present visual stimuli with the scanner's projector.
+    The lengths of the three *Psychopy experiments* should be:
```

Comment:
```suggestion
    The lengths of the four *Psychopy experiments* should be:
```
Should it be 4 with the addition of the dwi physiological/eye tracker acquisition ?

Reply:
Good catch!